### ISSUE TITLE
error parsing header "From"

### ISSUE THREAD

**Issue by rick8008** [versatica/JsSIP#560]
Hi, i'm having trouble on receive calls  and never goes to the callback "newRTCSession" . As i use the debug method, it shows this error :

**error parsing "From" header field with value ""XXXX" <sip:101@019f9ae299f5>;tag=7f749f4e-1a74-4b82-a38b-d1fa21b9d3a8"**

The invite is :
**INVITE sip:102@172.21.0.1:41874;transport=ws SIP/2.0
Via: SIP/2.0/WS 172.21.0.4:8088;rport;branch=z9hG4bKPj4be6ed2b-f0da-4b4c-a4e8-616ff1096e60;alias
From: "XXXXXXXX" <sip:101@019f9ae299f5>;tag=7f749f4e-1a74-4b82-a38b-d1fa21b9d3a8
To: <sip:102@172.21.0.1>
Contact: <sip:asterisk@019f9ae299f5:5060;transport=ws>
Call-ID: 97acb674-d024-44f5-95ac-d054c1d87436
CSeq: 17890 INVITE
Allow: OPTIONS, SUBSCRIBE, NOTIFY, PUBLISH, INVITE, ACK, BYE, CANCEL, UPDATE, PRACK, REGISTER, REFER, MESSAGE
Supported: 100rel, timer, replaces, norefersub
Session-Expires: 1800
Min-SE: 90
Max-Forwards: 70
User-Agent: Asterisk PBX 15.6.2
Content-Type: application/sdp
Content-Length:   916
v=0
o=- 606900586 606900586 IN IP4 172.21.0.4
s=Asterisk
c=IN IP4 172.21.0.4
t=0 0
a=msid-semantic:WMS *
a=group:BUNDLE audio-0
m=audio 10792 UDP/TLS/RTP/SAVPF 0 8 3 101
a=connection:new
a=setup:actpass
a=fingerprint:SHA-256 44:96:A1:30:52:22:8C:41:DC:35:7D:30:62:57:C2:89:CB:F9:A4:AE:17:D6:93:31:77:34:75:69:4F:2A:C9:C8
a=ice-ufrag:23188335017bef3e6676283f756874c1
a=ice-pwd:783b356e6f25e46c13b664075dd0d50d
a=candidate:Hac150004 1 UDP 2130706431 172.21.0.4 10792 typ host
a=candidate:Sb1dcac5f 1 UDP 1694498815 177.220.172.95 11591 typ srflx raddr 172.21.0.4 rport 10792
a=rtpmap:0 PCMU/8000
a=rtpmap:8 PCMA/8000
a=rtpmap:3 GSM/8000
a=rtpmap:101 telephone-event/8000
a=fmtp:101 0-16
a=ptime:20
a=maxptime:150
a=sendrecv
a=rtcp-mux
a=ssrc:1690756205 cname:cb284126-cf84-49a1-b0da-4035c5cecd9b
a=msid:2e152af6-bfde-4171-91d7-ebbbb68102da 51a2ff1e-3dcc-4014-962e-617c70e25fd7
a=mid:audio-0**

**Comment by rick8008** [comment]
Sorry. It was a copy and paste error.
This is the screenshoot:
![error](https://user-images.githubusercontent.com/5423302/51037469-76ddc080-1597-11e9-98df-a17ad84826b8.png)

**Comment by ibc** [comment]
The string after @ in the From header is not a valid IP or domain. So again not a bug in JsSIP. Fix your Asterisk setup to not place such a wrong URI in the From.

**Comment by rick8008** [comment]
Thank you, really seems to be error on asterisk return. Sorry :3

**Comment by rick8008** [comment]
I find what cause this error for me. i'm using asterisk inside a container and the hostname was the container id.

**Comment by ibc** [comment]
Yes. If Asterisk does not sanitize it, you must set a proper setting in your Asterisk conf to fix it.

**Comment by rick8008** [comment]
In my case i use "-h" in the run statement to fix it
